docs/guidelines/platforms/android/guideline_operable_android.md

```diff
@@ -78,16 +78,35 @@ When the user is navigating through the app **using the arrow keys on the keyboa
 
 Provide users enough time to read and use the content.
 
+### Timing Adjustable
+
+Ensure that users can adjust the timing of content that is displayed on the screen.
+
 *This guideline covers point 2.2.1 Timing Adjustable - Level A of the WCAG standard.*
 
 :white_check_mark: **Success criteria**
 
-All users should be able to interact with the content displayed on the screen, even if a time limit is defined for interaction with a specific view. Therefore, users should be able to turn off the defined time limit, adjust it or extend it.
+All users should be able to interact with the content displayed on the screen, even if a time limit is defined for interaction with a specific view. Therefore, users should be able to turn off the defined time limit, adjust it or extend it up to 10 times the intended time.
 
 - In case of session inactivity, it is recommended to notify the user that they are about to be signed off with the ability to extend that time limit. The notification could be displayed in the form of an AlertDialog or something similar. TalkBack service tells you about alerts and notifications so users using accessibility services would also be aware of the defined limit.
 
 - In the case of auto-updating content, it is recommended to allow the user to extend the defined time limit to at least ten times the length of the default setting so that they're able to process the displayed information.
```

Comment:
Do we have some nice examples of auto-updating content that does not comply as an exception, for additional clarity?

Reply:
I read into the criteria, but in general all auto-updating events are excepted since if we slow down the info update we would display wrong data.